File: python/disk.py
class disk:

    __type= ''
    __capacity = ''
    __price = 0

    def __init__(self, type = '', capacity = '', price = 0):
        self.__type = type
        self.__capacity = capacity
        self.__price = price
        
    #prosedur setter data
    def setType(self, type):
        self.__type = type
    
    def setCapacity(self, capacity):
        self.__capacity = capacity
    
    def setPrice(self, price):
        self.__price = price
    
    #prosedur getter data
    def getType(self):
        return self.__type
    
    def getCapacity(self):
        return self.__capacity

    def getPrice(self):
        return self.__price

File: C++/disk.cpp
#include <string>

using namespace std;

class disk
{
    private:
        //atribut
        string type;
        string capacity;
        int price;

    public:
        //konstruktor
        disk(){ }

        disk(string type, string capacity, int price){
            this->type = type;
            this->capacity = capacity;
            this->price = price;
        }

        //prosedur getter
        void setType(string type){
            this->type = type;
        }

        void setCapacity(string capacity){
            this->capacity = capacity;
        }

        void setPrice(int price){
            this->price = price;
        }

        //prosedur getter
        string getType(){
            return this->type;
        }

        string getCapacity(){
            return this->capacity;
        }

        int getPrice(){
            return this->price;
        }

        //destruktor
        ~disk(){ }
};

